test/support/constants.js

```diff
@@ -1,4 +1,4 @@
 Cypress.env({
-    FILE_ID_DOC_SANITY: '694517832310', // Read-only
+    FILE_ID_DOC_SANITY: '764564067308', // Read-only
```

Comment:
Uploaded a new file (Chicken Chicken) that doesn't have text at the center of the document to workaround cypress' overlap detection logic so that discoverability works